aten/src/THCUNN/common.h

```diff
@@ -8,9 +8,15 @@
 const int CUDA_NUM_THREADS = 1024;
 
 // CUDA: number of blocks for threads.
-inline int GET_BLOCKS(const int N)
+inline int GET_BLOCKS(const int64_t N)
 {
-  return (N + CUDA_NUM_THREADS - 1) / CUDA_NUM_THREADS;
+  // Round up division for positive number
```

Comment:
Technically, this works for N=0, so it's a round up division for non-negative number
```suggestion
  // Round up non-overflow division for non-negative number
```